docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,7 @@
+= Asciidoctor.js Documentation
+:uri-opal: https://opalrb.com
+
+Asciidoctor.js brings AsciiDoc to the JavaScript world!
+
+This project uses {uri-opal}[Opal] to transpile https://asciidoctor.org[Asciidoctor], a modern implementation of AsciiDoc, from Ruby to JavaScript to produce _asciidoctor.js_.
```

Comment:
I'm not sure about this introduction.
I don't think we should start off by mentioning Opal. I'm not saying that we should hide the fact that we are using Opal but we should probably focus on what Asciidoctor.js is and does and less on how Asciidoctor.js is built.